Example/Example/Images.xcassets/AppIcon.appiconset/Contents.json

```diff
@@ -139,8 +139,15 @@
       "size" : "44x44",
       "idiom" : "watch",
       "scale" : "2x",
-      "role" : "longLook",
-      "subtype" : "42mm"
+      "role" : "appLauncher",
+      "subtype" : "40mm"
+    },
+    {
+      "size" : "50x50",
+      "idiom" : "watch",
+      "scale" : "2x",
+      "role" : "appLauncher",
+      "subtype" : "44mm"
```

Comment:
looks like you added additional images? how does it affect builds?

Reply:
This is probably Xcode updating mandatory assets